docs/pages/docs/ci/github-actions.mdx

```diff
@@ -71,15 +71,25 @@ Create file called `.github/workflows/ci.yml` in your repository with the follow
 
           steps:
             - name: Check out code
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
+
+            - name: Setup Cache
+              uses: actions/cache@v3
               with:
-                fetch-depth: 2
+                path: |
+                  node_modules/.cache/turbo
+                  # if you use pnpm, uncomment the following lines
```

Comment:
The `pnpm` stuff should be dropped into the `pnpm` configuration later in this file.